src/components/cart.js

```diff
@@ -77,8 +78,8 @@ export default class Cart extends Component {
       const data = merge(lineItem, this.options.viewData);
       data.classes = this.classes;
       data.lineItemImage = this.imageForLineItem(data);
-      data.variantTitle = data.variant_title === 'Default Title' ? '' : data.variant_title;
-      data.formattedPrice = formatMoney(data.line_price, this.globalConfig.moneyFormat);
+      data.variantTitle = data.variant.title === 'Default Title' ? '' : data.variant.title;
+      data.formattedPrice = formatMoney(data.variant.price * data.quantity, this.globalConfig.moneyFormat);
```

Comment:
this should rely on the shop resource. Is that's what's driving `globalConfig` now?